core/templates/dev/head/pages/library/Library.js

```diff
@@ -250,19 +250,18 @@ oppia.controller('Library', [
     };
 
     // The carousels do not work when the width is 1 card long, so we need to
-    // handle this case discretely.
-    // Also preventing swiping past the first and last card.
+    // handle this case discretely. Also preventing swiping past the first
```

Comment:
`...handle this case discretely and also prevent swiping past the first and last card.`

Reply:
Done.